hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/ExponentialRetryPolicy.java

```diff
@@ -56,6 +56,12 @@ public class ExponentialRetryPolicy {
    */
   private static final double MAX_RANDOM_RATIO = 1.2;
 
+  /**
+   * All status codes less than http 100 signify error
+   * and should qualify for retry.
+   */
+  private static final int HTTP_CONTINUE = 100;
```

Comment:
you should only have one declaration of this and cross references; makes it easier to find usages in the IDE